packages/Stntuple/package.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+import os
+
+def sanitize_environments(env, *vars):
```

Comment:
It is unlikely this is necessary, I think you can just skip it

Reply:
thanks for pointing that out.  as you can guess, the recipe is just a copy&paste from somewhere else  :)